.gitmodules

```diff
@@ -31,3 +31,12 @@
 [submodule "external/alfons"]
 	path = external/alfons
 	url = https://github.com/hjanetzek/alfons
+[submodule "external/curl"]
+	path = external/curl
+	url = https://github.com/lygstate/curl.git
+[submodule "external/zlib"]
+	path = external/zlib
+	url = https://github.com/madler/zlib.git
+[submodule "external/angle"]
+	path = external/angle
+	url = https://github.com/lygstate/angle.git
```

Comment:
@lygstate I think it should be make no difference except that one does not have to checkout unnecessary submodules for a platform. The version to checkout is stored as "Subproject commit hash" as usual. And calling ```git submodule update``` afterwards should only update the initialized submodules to the current commit.

If curl builds against windows SSL I'm OK with it.